flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/generated/AggsFunctionWithWindowSize.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.runtime.generated;
+
+/**
+ * The interface to provide the ability to set window size for the aggregate functions that requires
+ * the size of current window to do calculation.
+ *
+ * <p>It is code generated to set the size of current window
+ */
+public interface AggsFunctionWithWindowSize {
+    void setWindowSize(int windowSize);
```

Comment:
I think it's fine to just put this method in `AggsHandleFunction` or `AggsHandleFunctionBase`. When generating handle for the non-window-sized function, the method can just be a no-op, like `retract`, `merge`, `reset` methods.